### PR TITLE
Update docs for git notifiers

### PR DESCRIPTION
**Pull request by phillebaba**

### CODE REVIEW

File: docs/guides/notifications.md [hunk 1]
```diff
@@ -103,19 +103,39 @@ When the verbosity is set to `info`, the controller will alert if:
...
 
-The resulting status will contain information from the event in the format `{{ .Kind }}/{{ .Name }} - {{ .Reason }}`.
+The first image is an example of how it may look like in GitHub while the one below is an example for GitLab.
 ![github commit status](../diagrams/github-commit-status.png)
+![gitlab commit status](../diagrams/gitlab-commit-status.png)
 
-It is important to note that the referenced provider needs to refer to the
-same GitHub repository as the Kustomization originates from. If these do
-not match the notification will fail as the commit id will not be present.
+Currently the provider will only work with Alerts for Kustomization resources as the events have to be linked with a
```
Comment (phillebaba) [inline docs/guides/notifications.md:117]: Should this be included?

File: docs/guides/notifications.md [hunk 2]
```diff
@@ -144,16 +164,5 @@ spec:
...
-apiVersion: v1
-kind: Secret
-metadata:
-  name: github
-  namespace: gitops-system
-data:
-  token: <token>
-```
-
+### Health checks
```
Comment (phillebaba) [inline docs/guides/notifications.md:167]: Need to add some examples how notifications behave when a Kustomization goes from healthy to unhealthy.

File: docs/guides/notifications.md [hunk 3]
```diff
@@ -103,19 +103,39 @@ When the verbosity is set to `info`, the controller will alert if:
...
-state of a Kustomization resource, will update the
-[commit status](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks) for the currently reconciled commit id.
+The `github` and `gitlab` provider are slightly different to the other chat providers. These providers will
+link an event back to its source by writing a commit status event to the repository. For more information about how a
+commit status works, refer to the [GitHub](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-status-checks)
+or [GitLab](https://docs.gitlab.com/ee/api/commits.html) documentation.
 
-The resulting status will contain information from the event in the format `{{ .Kind }}/{{ .Name }} - {{ .Reason }}`.
+The first image is an example of how it may look like in GitHub while the one below is an example for GitLab.
 ![github commit status](../diagrams/github-commit-status.png)
```
Comment (phillebaba) [inline docs/guides/notifications.md:114]: Need to update the image used.

File: docs/guides/notifications.md (hunk 1 above)
Comment (stefanprodan) [reply docs/guides/notifications.md:117]: Yes this look ok to me 👍

File: docs/guides/notifications.md (hunk 3 above)
Comment (stefanprodan) [reply docs/guides/notifications.md:114]: Let's update the image in a followup PR.

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @phillebaba 🏅

**Pull request merged**